### PR TITLE
write app version

### PR DESCRIPTION
**Pull request by ubozov**
Resolves BV-1047.

### CODE REVIEW

File: data/data.go [hunk 1]
```diff
@@ -11,6 +11,15 @@ import (
 	_ "github.com/lib/pq"
 )
 
+const (
+	versionKey    = `system.version.app`
+	selectVersion = `SELECT value FROM settings WHERE key = $1;`
```
Comment (dzeckelev) [inline data/data.go]: We need these constants? Will we use this more than once?

File: data/data.go [hunk 2]
```diff
@@ -11,6 +11,15 @@ import (
...
+	insertVersion = `INSERT INTO
+		settings (key, value, permissions, description, name)
+		VALUES ($1, $2, 1, 'Version of application', 'app version');`
```
Comment (dzeckelev) [inline data/data.go]: why after $2 goes $1?

File: data/data.go (hunk 1 above)
Comment (ubozov) [reply data/data.go]: updated, YAGNI

File: data/data.go (hunk 2 above)
Comment (ubozov) [reply data/data.go]: after $2 goes 1 (not $1)

Comment (dzeckelev) [reply data/data.go]: `UPDATE settings SET value = $2 WHERE key = $1;`

Comment (ubozov) [reply data/data.go]: it is the number of variables
```
sqlStatement = `UPDATE settings SET value = $2 WHERE key = $1;`
_, err = conn.Exec(sqlStatement, versionKey, version)
```

Comment (dzeckelev) [reply data/data.go]: I do not understand. What is so bad?
```
sqlStatement = `UPDATE settings SET value = $1 WHERE key = $2;`
_, err = conn.Exec(sqlStatement, version, versionKey)
```

**Pull request merged**